src/main/java/cr/pernix/dashboard/services/CompanyService.java

```diff
@@ -38,7 +39,7 @@ public static synchronized CompanyService getInstance() {
         transaction.begin();
         if (transaction.getStatus().equals(TransactionStatus.NOT_ACTIVE))
             LOGGER.debug(" >>> Transaction close.");
-        Query query = session.createQuery("from Company");
+        Query query = session.createQuery("from Company c where c.active = true");
```

Comment:
You should add another method to get the inactive companies, and  activate one of thouse. Someone could inactive a company by error